airflow/hooks/postgres_hook.py

```diff
@@ -23,6 +23,7 @@
 from contextlib import closing
 
 from airflow.hooks.dbapi_hook import DbApiHook
+from airflow.contrib.hooks.aws_hook import AwsHook
```

Comment:
We should probably make this import conditional (i.e. not at the top-level but inside get_iam_token) so that the Pg hook doesn't require Boto for those people not using this feature.